src/SerializeAST.h

```diff
@@ -10,10 +10,21 @@
 #define DRAFTER_SERIALIZEAST_H
 
 #include "Serialize.h"
+#include "SectionProcessor.h" // from Blueprint
 
 namespace drafter {
 
-    sos::Object WrapBlueprint(const snowcrash::Blueprint& blueprint, const ASTType astType, bool expand = false);
+    struct WrappingContext {
+        const ASTType astType;
+        const bool expandMSON;
+        const bool exportSourceMap;
+
+        WrappingContext(const ASTType astType, const bool expandMSON, const bool exportSourceMap) 
+            : astType(astType), expandMSON(expandMSON), exportSourceMap(exportSourceMap) {
+        }
+    };
```

Comment:
I don't agree. Extending the `enum snowcrash::BlueprintParserOption` as `enum drafter::BlueprintWrapOption` would be better instead of introducing a different way of doing this with `WrappingContext`.